r/R/dplyr-across.R

```diff
@@ -175,3 +162,13 @@ across_setup <- function(cols, fns, names, .caller_env, mask, inline = FALSE) {
 across_glue_mask <- function(.col, .fn, .caller_env) {
   env(.caller_env, .col = .col, .fn = .fn, col = .col, fn = .fn)
 }
+
+quosure_from_func_and_col <- function(func, col) {
```

Comment:
Can you add comments explaining why this is necessary?

Does `dplyr` do this too?

Reply:
Now refactored it to do it the dplyr way!